docs/technical_reference_guide/3a_ResStockInputs.tex

```diff
@@ -1915,14 +1928,10 @@ \subsubsection{Insulation Rim Joist}\label{insulation_rim_joist}
 \end{itemize}
 
 \paragraph{Option(s)}
-ResStock uses four different levels of rim-joist insulation as well as a ``None'' flag for homes without rim joists. These options map directly to the foundation wall insulation levels. The ResStock argument for
-\texttt{rim\_joist\_assembly\_r} is always set to \textit{auto}, and
-\texttt{rim\_joist\_continuous\_interior\_r} and
-\texttt{rim\_joist\_assembly\_interior\_r} are both set to 0 for all options.
```

Comment:
Lots of situations like this where bug chunks of documentation could be deleted. That's why the PR reduces the documentation by ~500 lines.